nvsvocprez/app.py

```diff
@@ -1810,8 +1811,13 @@ def _render_sdo_rdf(self):
         )
         return self._render_sparql_response_rdf(sparql_construct(q, self.mediatype))
 
-    def _render_puv_rdf(self):
-        q = """
+    def _render_profile_rdf(self):
+        exclude_filters = ""
+        exclude_profiles = [alt["url"] for alt in self.alt_profiles.values() if alt["token"] != self.profile]
+        for ep in exclude_profiles:
+            exclude_filters+= f'FILTER (!STRSTARTS(STR(?p), \"{ep}\"))\n'
```

Comment:
Are the escaped quotes here needed? I think un-escaped should be fine as you've wrapped the string in single quotes.

Reply:
Yep, that's a relic from when there was alt["url"] instead of ep